modules/preview-react/menu/lib/Menu.tsx

```diff
@@ -211,7 +212,7 @@ export default class Menu extends React.Component<MenuProps, MenuState> {
           isShortcut = true;
           const nextIndex = this.state.selectedItemIndex + direction;
           nextSelectedIndex =
-            nextIndex < 0 ? lastItem : nextIndex >= itemCount ? firstItem : nextIndex;
+            nextIndex < 0 ? lastItem : nextIndex >= interactiveItemCount ? firstItem : nextIndex;
```

Comment:
To go along with suggestion on line 189:
```suggestion
            nextIndex < 0 ? lastItemIndex : nextIndex >= interactiveItemCount ? firstItem : nextIndex;
```